### PR TITLE
Added a new feature branch 'copo_metadata' to solve issue #115

### PR DESCRIPTION
**Pull request by SandraBabirye**
- [ ] Added a python script that parses the json file that is downloaded from the COPO datasource to extract metadata from it
- [ ] Extracted only relevant information for the genome note from the parsed file
- [ ] Add the list of files in bin/combine_parsed_data.py ie " ("COPO_BIOSAMPLE", "copo_biosample_wgs_file"),
    ("COPO_BIOSAMPLE_HIC", "copo_biosample_hic_file"),
    ("COPO_BIOSAMPLE_RNA", "copo_biosample_rna_file"),"
- [ ] Make sure your code lints (`nf-core lint`).
- [ ] Ensure the test suite passes (`nextflow run . -profile test,docker --outdir <OUTDIR>`).

### CODE REVIEW

File: modules/local/parse_metadata.nf
Comment (BethYates) [inline modules/local/parse_metadata.nf:1]: This change isn't necessary, the datasource is COPO, but the biosample types are still just "WGS", "HIC" or "RNA"

Comment (SandraBabirye) [reply modules/local/parse_metadata.nf:1]: Oh okay I have removed that @BethYates . Thank you

File: bin/parse_json_copo_biosample.py [hunk 1]
```diff
@@ -0,0 +1,123 @@
...
+    sys.exit(1)
+
+
+def parse_json(file_in, file_out):
+    biosample_type = "COPO"
+    with open(file_in, "r") as json_file:
+        data = json.load(json_file)
+
+    param_list = []
+    record = data["records"][0]  # Get the single record
```
Comment (BethYates) [inline bin/parse_json_copo_biosample.py:74]: "records" is not a key in the copo json file instead use "data" so line 74 should be 
` record = data["data"][0]  # Get the single record`

File: bin/parse_json_copo_biosample.py [hunk 2]
```diff
@@ -0,0 +1,123 @@
...
+
+    if len(data["records"]) != 1:
```
Comment (BethYates) [inline bin/parse_json_copo_biosample.py:76]: In this case the json file tells us how many results are found using the key "number_found" so line 76 can be replaced with 
`if data["number_found"] != 1:`

File: bin/parse_json_copo_biosample.py (hunk 1 above)
Comment (SandraBabirye) [reply bin/parse_json_copo_biosample.py]: I have edited that. Thank you

File: bin/combine_parsed_data.py
Comment (BethYates) [inline bin/combine_parsed_data.py:1]: You will also need to add the following lines to the argument list
```
parser.add_argument("--copo_biosample_wgs_file", help="Input parsed COPO genomic biosample file.", required=False)
parser.add_argument("--copo_biosample_hic_file", help="Input parsed COPO HiC biosample file.", required=False)
parser.add_argument("--copo_biosample_rna_file", help="Input parsed COPO RNASeq biosample file.", required=False)
```

**Pull request merged**